README.md

```diff
@@ -407,6 +407,12 @@ For more information see [Plugins](https://github.com/laurent22/joplin/blob/dev/
 
 Joplin implements the SQLite Full Text Search (FTS4) extension. It means the content of all the notes is indexed in real time and search queries return results very fast. Both [Simple FTS Queries](https://www.sqlite.org/fts3.html#simple_fts_queries) and [Full-Text Index Queries](https://www.sqlite.org/fts3.html#full_text_index_queries) are supported. See below for the list of supported queries:
 
+One caveat of SQLite FTS is that it does not support languages which do not use Latin word boundaries (spaces, tabs, punctuation). To solve this issue, Joplin has a custom search mode, that does not use FTS, but still has all of its features (multi term search, filters, etc.). Its only drawback is that it can get slow on larger note collections. This search mode is currently enabled if one of the following languages are detected:
```

Comment:
Probably should be "Its only drawback" => "One of its drawback". Also please mention that sorting is less accurate too (since for FTS we can use BM25). And how about prefix queries? Do they work with this new mode?